gpkit/constraints/bounded.py

```diff
@@ -25,12 +24,14 @@ def varkey_bounds(varkeys, lower, upper):
     constraints = []
     for varkey in varkeys:
         variable = Variable(**varkey.descr)
-        units = varkey.units if isinstance(varkey.units, Quantity) else 1
+        if variable.units:
+            variable.hmap.units = None
+            variable.units = None
```

Comment:
The old way was much much slower :( this is ~1s faster on SPAircraft.

Because his is a monomial, not a varkey, setting those two things to None is exactly equivalent to dividing by its units, though I think I'll add a unit test to show that!